lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1099,7 +1124,7 @@ def test_create_order_success(self):
             photo_id_image=self.IMAGE_DATA
         )
         json_response = json.loads(response.content)
-        self.assertIsNotNone(json_response.get('orderNumber'))
+        self.assertIsNotNone(json_response['payment_form_data'].get('orderNumber'))  # TODO not canonical
```

Comment:
I might be confused about the behavior being tested here. Is this meant to be data returned by the ecommerce API? The payment form data sent back by the ecommerce API doesn't include an `orderNumber` key.